recipes/search_ncbi/meta.yaml

```
@@ -0,0 +1,45 @@
+  host:  
+    - python >=3.7  
+    - pip  
+  run:  
+    - python >=3.7  
+    - biopython >=1.78  
+    - pandas >=1.3.0  
+    - tqdm >=4.66.3  
+    - xmltodict >=0.13.0
+    - requests >=2.32.0
+
+test: 
```

yamllint

[error] trailing spaces

(trailing-spaces)